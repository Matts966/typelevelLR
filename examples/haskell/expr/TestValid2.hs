
import Expr

main :: IO ()
main = print $ begin num 1 add num 2 add num 3 end
