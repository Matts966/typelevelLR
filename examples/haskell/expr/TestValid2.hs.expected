Add (Add (TToE (FToT (Num 1))) (FToT (Num 2))) (FToT (Num 3))
